src/v/wasm/fwd.h

```diff
@@ -11,12 +11,16 @@
 
 #pragma once
 
+#include "pandaproxy/schema_registry/api.h"
```

Comment:
What symbols do we need from here? Can we move this to the downstream includes that need this?